src/app/modules/main/profile_section/wallet/networks/view.nim

```diff
@@ -42,29 +44,47 @@ QtObject:
     self.areTestNetworksEnabled = not self.areTestNetworksEnabled
     self.areTestNetworksEnabledChanged()
 
-  proc networksChanged*(self: View) {.signal.}
-
-  proc getNetworks(self: View): QVariant {.slot.} =
-    return newQVariant(self.networks)
-
-  QtProperty[QVariant] networks:
-    read = getNetworks
-    notify = networksChanged
-
-  proc load*(self: View, networks: seq[NetworkDto]) =
-    var items: seq[Item] = @[]
-    for n in networks:
-      items.add(initItem(
-        n.chainId,
-        n.layer,
-        n.chainName,
-        n.iconUrl,
-        n.shortName,
-        n.chainColor
-      ))
-
-    self.networks.setItems(items)
+  proc combinedNetworksChanged*(self: View) {.signal.}
+  proc getCombinedNetworks(self: View): QVariant {.slot.} =
+    return newQVariant(self.combinedNetworks)
+  QtProperty[QVariant] combinedNetworks:
+    read = getCombinedNetworks
+    notify = combinedNetworksChanged
+
+  proc load*(self: View, combinedNetworks: seq[CombinedNetworkDto]) =
+    var combinedItems: seq[CombinedItem] = @[]
```

Comment:
@Khushboo-dev-cpp this should go to `module.nim`, since this is business logic.
Also I've noticed that in `module.nim` file `refreshNetworks` function calls `view.load` function, which is somehow unexpected. So just call `view.load` from `module.load` and then `refreshNetworks` from `module.load` where `refreshNetworks` will prepare items, get necessary models from view and set items to them (what's actually at this moment done in `view.load`).

Reply:
done @saledjenic